### ISSUE TITLE
add dry run option

### ISSUE THREAD

**Issue by perfecto25** [perfecto25/poni#1]
1. add dry run option, to show potential sync before doing actual sync
2. allow global remote_user, priv_key options ie

```
remote_host: abc
remote_path: /home/jsmith/
remote_user: jsmith
priv_key: /home/jsmith/.ssh/id_rsa
recurse: true
interval: 15

sync:
  /home/jsmith/dir1: []
  /home/jsmith/dir2: []
  /home/blah/dir3:
    remote_host: xyz
    remote_user: blah
    priv_key: /home/blah/.ssh/id_rsa
    recurse: false
    interval: 60

```

first 2 syncs should inherit global options, 3rd should use unique options

**Comment by perfecto25** [comment]
global values are available in 0.1.2

**Comment by perfecto25** [comment]
added "simulate", which is dry run
